integration-tests/common/src/test/kotlin/me/tatarka/inject/test/FunctionTest.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("unused")
```

Comment:
I'd be fine with getting rid of unused variables in tests, but it's a pretty minor thing.

Reply:
I replaced the file supressions with individual ones.
The thing is that they are not actually unused but necessary for the test setup.